src/ImpurityDensities/ThermalDiffusionLithiumDensity.jl

```diff
@@ -0,0 +1,68 @@
+"""
+    struct ThermalDiffusionLithiumDensity{T <: SSDFloat} <: AbstractImpurityDensity{T}
+
+Lithium impurity density model
+
+ref: [Dai _et al._ (2023)](https://doi.org/10.1016/j.apradiso.2022.110638)
+ 
+## Fields
+* `Li_annealing_temperature::T`: lithium annealing temperature when the lithium is diffused into the crystal. The default value is 623 K.
+* `Li_annealing_time::T`: lithium annealing time. The default value is 18 minutes.
+* `calculate_depth2surface::Function`: the function for describing the depth to surface
+* `Li_surface::T`: the lithium concentration in the surface
```

Comment:
To me `Li_surface` would be something I would call a geometric object, not the actual doping concentration..